MCServer/Prefabs/Villages/AlchemistVillage.cubeset

```diff
@@ -0,0 +1,3647 @@
+
+-- AlchemistVillage.cubeset
+
+-- Defines the prefabs in the group AlchemistVillage
+
+-- NOTE: This file has been generated automatically by GalExport!
+-- Any manual changes will be overwritten by the next automatic export!
+
+
+
+
+Cubeset =
+{
+	Metadata =
+	{
+		CubesetFormatVersion = 1,
+		ExportDate = "2015-06-20 10:17:14",
+		["AllowedBiomes"] = "Desert, DesertM",
+		["IntendedUse"] = "Village",
```

Comment:
Yeah, I know they are the same, but I just noticed that they were both used here ;)